### PR TITLE
json-table 4.3.3 (new formula)

### PR DESCRIPTION
**Pull request by alandipert**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by alandipert** [comment]
Hello, thank you in advance for giving my PR a look. Please let me know if there are ways I can improve it.

File: Formula/json-table.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class JsonTable < Formula
+  desc "Transform nested JSON data into tabular data in the shell."
+  homepage "https://github.com/micha/json-table"
+  url "https://github.com/micha/json-table/archive/4.3.3.tar.gz"
+  sha256 "0ab7bb2a705ad3399132060b30b32903762473ff79b5a6e6f52f086e507b0911"
+  head "https://github.com/micha/json-table.git"
+
+  def install
+    system "make"
```
Comment (ilovezfs) [inline Formula/json-table.rb]: does it fail without this line?

File: Formula/json-table.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class JsonTable < Formula
+  desc "Transform nested JSON data into tabular data in the shell."
```
Comment (ilovezfs) [inline Formula/json-table.rb]: period should be removed

**Comment by alandipert** [comment]
@ilovezfs thanks for the feedback. I pushed updates.

File: Formula/json-table.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/micha/json-table/archive/4.3.3.tar.gz"
+  sha256 "0ab7bb2a705ad3399132060b30b32903762473ff79b5a6e6f52f086e507b0911"
+  head "https://github.com/micha/json-table.git"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "bash", "-c", "[ $(echo '{\"x\":{\"y\":[1,2,3]}}' | #{bin}/jt x y % | tail -n1) == 3 ]"
```
Comment (ilovezfs) [inline Formula/json-table.rb]: This shouldn't need to use bash for the test.

Comment (alandipert) [reply Formula/json-table.rb]: @ilovezfs the project has a `make test`, should I use that instead? Or just do something like `jt -h`

Comment (ilovezfs) [reply Formula/json-table.rb]: `make test` isn't possible in the test block. I mean like
```
  test do
    output = pipe_output("#{bin}/jt x y %", '{"x":{"y":[1,2,3]}}', 0)
    assert_equal "3", output.lines.last.chomp
  end
```
though a test that's a bit more involved would be good. (See `jose` and `vim` for some examples)

Comment (alandipert) [reply Formula/json-table.rb]: Ah, I see what you mean, thanks for clarifying. I incorporated your suggestion.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
Thanks @alandipert!

**Pull request closed without merging**